plugins/VersionUpgrade/VersionUpgrade33to34/VersionUpgrade33to34.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+import configparser #To parse preference files.
+import io #To serialise the preference files afterwards.
+
+from UM.VersionUpgrade import VersionUpgrade #We're inheriting from this.
+
+
+##  Upgrades configurations from the state they were in at version 3.2 to the
+#   state they should be in at version 3.3.
```

Comment:
Update copy-pasted documentation: Version 3.3 to 3.4.

Reply:
Fixed in ad3317a99